CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Networking/GiniCaptureNetworkService.swift

```diff
@@ -73,6 +77,25 @@ public extension GiniCaptureNetworkService {
                       completion: @escaping DocumentPagePreviewCompletion) {
         // Default implementation is empty
     }
+    
+    /**
+     *  Sends feedback for a document based on the provided document ID, updated extractions, and updated compound extractions.
+     *
+     * - Parameter documentId:                  The unique identifier of the document for which feedback is being sent.
+     * - Parameter updatedExtractions:          An array of updated extractions to be included in the feedback.
+     * - Parameter updatedCompoundExtractions:  A dictionary of updated compound extractions to be included in the feedback.
+     * - Parameter completion:                  A completion callback that returns a `Result<Void, GiniError>`.
+     *                                          On success, it returns `Void`. On failure, it returns an error.
+     *
+     * - Note: Prefer using this method over `sendFeedback(document:updatedExtractions:updatedCompoundExtractions:completion:)`
```

Comment:
[suggestion] What about this:
```
 - Note:
   This method is preferred over the `sendFeedback(document:updatedExtractions:updatedCompoundExtractions:completion:)`, as it requires only the `documentId` instead of the entire document object, making it more lightweight and flexible.
   ```